wiki/Welcome.md

```diff
@@ -0,0 +1,49 @@
+## Welcome do DragonProxy WIKI
+
+### 1. Presentation
+DragonProxy aim to be the first Minecraft PE <=> PC bridge, based on protocole translation.<br />
+With this proxy, PE players (W10, Android / IOS) can join PC servers.<br />
+This is probably the future of Minecraft servers, as Microsoft does not provide featured PE server publicly !
```

Comment:
Lol you realise featured servers just use regular third party server softwares like PocketMine-MP

Reply:
I was talking about server core, PocketMine-MP is "fan-made" and does not provide a good server as the java edition.